Formula/osslsigncode.rb

```diff
@@ -14,22 +14,42 @@ class Osslsigncode < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "2549903650fdd0189c1083a177af1c789364758080828b8ea6cf46c376ed6d65"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "openssl@1.1"
+  depends_on "cmake" => :build
+  depends_on "openssl@3"
 
   uses_from_macos "curl"
 
+  # Fix file INSTALL issue and install completion as part of `install` step
+  patch :DATA
+
   def install
-    system "./bootstrap"
-    system "./configure", *std_configure_args
-    system "make", "install"
+    # Fixing libcurl linker issue with `curl_version`
+    curl_lib = OS.mac? ? MacOS.sdk_path_if_needed/"usr/lib" : Formula["curl"].opt_lib
```

Comment:
`MacOS.sdk_path_if_needed` can be `nil`. Can either modify conditional or use string interpolation `#{MacOS.sdk_path_if_needed}/usr/lib"`

Reply:
this is actually adopted from the formula, what is your proposed fix?